README.md

```diff
@@ -9,76 +9,248 @@ Low-level R library bindings
 
 ## Installation
 
-To build this library and test that everything is working as expected, run the following two commands in the library's top-level directory:
+The recommended way to build this library is to use precomputed bindings, which are available for `Linux`, `MacOS`, and `Windows` (`32`- and `64`-bit).
 
-```bash
-cargo build
-cargo test
-```
+Alternatively, the library can be built from source, in which case it invokes `bindgen` crate, which has extra platform-specific dependencies (including `msys2` for `Windows`).
 
-The build script has the following two dependencies:
 
-1. [R](https://cran.r-project.org/): It needs to be installed and available in the search path.
-2. [libclang](https://clang.llvm.org/docs/Tooling.html): Depending on your operating system, you may need to set the `LIBCLANG_PATH` environment variable or add `llvm-config` to your search path.
+## Using precomputed bindings (recommended)
 
+Two components are required to build the library:
+1. [R](https://cran.r-project.org/): It needs to be installed and available in the search path.
+2. [rust](https://www.rust-lang.org/learn/get-started): It is recommended to install `rust` using `rustup`; search path should include `rust` binaries.
+
+
+When building for `Windows`, the default host shall be `stable-msvc` and special `rust` targets should be added for compatibility with `R`:
```

Comment:
Sorry, one last nitpick: "shall" sounds very old-fashioned and biblical. It's probably better to either write "should" or, if you want to make a stronger statement, "must".

Reply:
Done